cmd/stork/stork.go

```diff
@@ -354,6 +354,20 @@ func run(c *cli.Context) {
 	if err := apis.AddToScheme(mgr.GetScheme()); err != nil {
 		log.Fatalf("Setup scheme failed for stork resources: %v", err)
 	}
+	// Registering application registration CRDs explicitly to use
+	// shared informer cache for caching application-controller CRs
+	if c.Bool("application-controller") {
```

Comment:
Lets create the CRDs regardless of whether application controller is enabled or not, since the cache's APIs are not guarded by this flag.

Reply:
Agree. Updated. Will merge and cherry-pick after successful CBT run.